### DISCUSSION TITLE
Manual/UnityWebRequest-SendingForm

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#671] [Notes]
# Manual/UnityWebRequest-SendingForm

https://docs.unity3d.com/Manual/UnityWebRequest-SendingForm.html

**Comment by StamatisP** [comment]
The documentation is wrong, using MultipartFormDataSection like `new MultipartFormDataSection("field1=foo&field2=bar")` is incorrect. Instead use `new MultipartFormDataSection("field1", "foo")` and `new MultipartFormDataSection("field2", "bar")`.
I have an example here: https://gist.github.com/StamatisP/a95e0e4af68383ca993b6d33b914268e
